addons/account_edi_ubl_cii/models/account_edi_format.py

```diff
@@ -132,7 +132,7 @@ def _post_invoice_edi(self, invoices):
             if self.code not in ['facturx_1_0_05', 'efff_1', 'nlcius_1']:
                 attachment_create_vals.update({'res_id': invoice.id, 'res_model': 'account.move'})
 
-            attachment = self.env['ir.attachment'].create(attachment_create_vals)
+            attachment = self.env['ir.attachment'].with_user(SUPERUSER_ID).create(attachment_create_vals)
```

Comment:
Why not a simple `sudo` so that we keep the right user?
```suggestion
            attachment = self.env['ir.attachment'].sudo().create(attachment_create_vals)
```

Reply:
Oh, `with_user` removes the sudo... :confused: